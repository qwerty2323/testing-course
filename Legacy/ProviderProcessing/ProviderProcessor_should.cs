using System;
using ApprovalTests;
using ApprovalTests.Core;
using ApprovalTests.Namers;
using ApprovalTests.Reporters;
using FakeItEasy;
using NUnit.Framework;
using Ploeh.AutoFixture;
using StatePrinter;

namespace Legacy.ProviderProcessing
{
	[TestFixture]
	[UseReporter(typeof(DiffReporter))]
	public class ProviderProcessor_should
	{
		[Test]
		public void SaveDataForNewProvider()
		{

		}

		[Test]
		public void IgnoreOutdatedData()
		{

		}
	}
}