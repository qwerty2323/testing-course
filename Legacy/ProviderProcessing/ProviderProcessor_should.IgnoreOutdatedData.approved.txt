new ProcessReport()
{
    Success = False
    Error = "Outdated data"
    ProductErrors = new String[]()
}